.github/workflow/nodejs.yml

```diff
@@ -0,0 +1,27 @@
+name: Node-CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 13.x]
```

Comment:
This doesn't match announced version support: https://github.com/nfarina/homebridge/blob/master/package.json#L24

Reply:
This is just like [hap-nodejs](https://github.com/KhaosT/HAP-NodeJS/raw/master/.github/workflows/nodejs.yml)

Should 4.x be added?

Reply:
Looks like we better wait for this one to be decided on – https://github.com/nfarina/homebridge/issues/2401

Meanwhile, keeping those three should be fine, probably.